Aptoide/aptoide-client-v8
# [ASV-401] More button for app coins ads bundle

**What does this PR do?**

This pull request is the implementation of the more button for the Get Rewarded With AppCoins bundle. It should open a fragment with a list of apps that have appcoins rewards to the users.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] GetRewardAppCoinsAppsFragment.java
- [ ] GetRewardAppCoinsAppsWidget.java
- [ ] GetRewardAppCoinsAppsDisplayable.java

**How should this be manually tested?**

Open app, press more on the get rewarded with appcoins bundle, see list of apps.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-401](https://aptoide.atlassian.net/browse/ASV-401)

**Questions:**


**Code Review Checklist**
